microsoft/terminal
# Copy CWD action

## Summary of the Pull Request
Added action for copy CWD

 
## References
Require configuration: https://docs.microsoft.com/en-us/windows/terminal/tutorials/new-tab-same-directory


## PR Checklist
* [x] Closes #12860
* [x] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #12860


## Detailed Description of the Pull Request / Additional comments


## Validation Steps Performed
- Ensure to have shell configured to update CWD: https://docs.microsoft.com/en-us/windows/terminal/tutorials/new-tab-same-directory
- Terminal > Settings > Actions: assign a shortcut to "Copy CWD" action
- Navigate directories and copy the CWD with the configured action
